integrationtesting/fixtures/src/main/java/org/opendcs/fixtures/configurations/cwms/CwmsOracleConfiguration.java

```diff
@@ -149,6 +153,33 @@ public void start(SystemExit exit, EnvironmentVariables environment, SystemPrope
         }
     }
 
+    /**
+     * Load location level test data for integration tests
+     */
+    private void loadLocationLevelTestData()
```

Comment:
This is better done in the test itself or at least the test class.

This section is for baseline data, which is only the minimum required to even start the applications and have them behave correctly.